scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -30,13 +30,17 @@ def remove_files(files):
 
 
 def cpuflags():
-    cpu_flags = readfile("{0}/cpu-default.xml".format(TEMPLATE_DIR))
-    if (subprocess.call(["grep", "-q", "flags.* npt", "/proc/cpuinfo"]) == 0):
```

Comment:
ah, because I was not on tip of master, where you dropped the other use. ok then. done.